rllib/utils/replay_buffers/replay_buffer.py

```diff
@@ -57,6 +57,53 @@ def warn_replay_capacity(*, item: SampleBatchType, num_items: int) -> None:
 
 @DeveloperAPI
 class ReplayBuffer(ParallelIteratorWorker):
+    """The lowest-level replay buffer interface used by RLlib.
```

Comment:
Awesome.

Reply:
Could we add a TODO to remove the `ParallelIteratorWorker` once we no longer support the `execution_plan` API?